### ISSUE TITLE
test_models.py out of sync with models.py?

### ISSUE THREAD

**Issue by ejcramer** [ripeta/repeat-aft#129]
With the recent changes in my local setup for repeat-aft (Latest commit a9a629a  11 hours ago) and repeat-plugins (Latest commit b43afcd  15 hours ago), when I run "./setup.py test", I get:

```
from tensorflow.python.keras.models import load_model
ImportError: No module named tensorflow.python.keras.models
```

And I checked the tensorflow version installed:
```
(env) ellens-mbp:repeat-aft ejc$ pip3 list | grep tensorflow
tensorflow                1.8.0 
```

But I can run "./setup.py build" and the service successfully otherwise.  It usually just fails on the plugins that are not in the unit tests yet. I don't remember seeing this before, but it the error could have been there for a while.

**Comment by ejcramer** [comment]
Hi Nick, Could you let me know when/if this is fixed.  I'd like to merge the recent changes with the master branch, but it won't pass the CI Travis testing until this works.  It's not urgent, but I'd like to do it before there's any major merge conflicts.  Thanks!

**Comment by DragonWriter** [comment]
When I run "./setup.py test" on my desktop's vm I also get the markdown error.
In looking it up I found this issue: https://github.com/encode/django-rest-framework/issues/6203 
in which it seems to be a potential compatibility issue with Markdown 3.0 which is what it currently has installed. 
Testing it on my laptop, which has Markdown 2.6.11 it ran without a problem. So until the issue with markdown 3.0 is resolved we probably want to add 2.6.11 a strict requirement. While that succeeded it ultimately did lead to one error in the article_title plugin due to the fact that I am assuming the text has a second line (which I guess in the test scenario it does not), but that should be an easy fix for the next repeat-plugins commit. 

I personally did not get a tensorflow version error in either of my setups though.

**Comment by DragonWriter** [comment]
Changing the Markdown version on my desktop from 3.0 to 2.6.11 resolved that error (mentioned by Elly in  #130).  Although this works as a good simple workaround, we may want to keep an eye on future versions of markdown and the issue linked above. 
Similarly I have added a line to article_title that resolves that error that will be in the next repeat-plugins commit.
